db/migrate/20160309223941_add_process_to_miq_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddProcessToMiqRequests < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_requests, :process, :boolean
```

Comment:
Oh haha...there you go.

Do you need a data migration to default everything to true?